backend/src/main/java/com/thoughtworks/aceleradora/controller/EstimateController.java

```diff
@@ -29,7 +29,13 @@ public EstimateController(EstimateService estimateService) {
     @PostMapping
     @ResponseStatus(HttpStatus.CREATED)
     public EstimateResponse create(@RequestBody EstimateRequest estimateRequest) {
-        return from(estimateService.create(estimateRequest));
+        //FIXME remover apos backend estar pronto
+        //return EstimateResponse.builder()
+               // .estimateCode(10)
+               // .build();
+       //return from(estimateService.create(estimateRequest));
+        estimateService.create(estimateRequest);
+       return EstimateResponse.builder().build();
```

Comment:
Aqui poderiamos passar de voltya o codigo do Estimate no banco não? timpo assim:
```suggestion
      Estimate estimateEntity = estimateService.create(estimateRequest);
       return EstimateResponse.builder()
               .estimateCode(estimateEntity.getId())
               .build();
       
```

Reply:
Resolvido @ldiasrs ^^